changelog.md

```diff
@@ -68,6 +68,8 @@
 - Added `minIndex`, `maxIndex` and `unzip` to the `sequtils` module.
 - Added `os.isRelativeTo` to tell whether a path is relative to another
 - Added `resetOutputFormatters` to `unittest`
+- added `os.relativePath(path)`, and made `os.relativePath` more consistent when mixing absolute
```

Comment:
What's the point of this new ``os.relativePath``? To be relative against getCurrentDir()? Please remove it.

Reply:
I'll debate this in some subsequent PR, for now I've removed it from this PR to unblock this